yoti_python_sdk/doc_scan/session/create/filter/orthogonal_restrictions_filter.py

```diff
@@ -0,0 +1,178 @@
+        self.__country_codes = country_codes
+
+    @property
+    def inclusion(self):
+        """
+        Returns the inclusion for the country restriction
+
+        :return: the inclusion
+        :rtype: str
+        """
+        return self.__inclusion
+
+    @property
+    def country_codes(self):
+        """
+        Returns the country codes for the restriction
+
+        :return: the country codes
+        :rtype: list[str]
+        """
+        return self.__country_codes
+
+    def to_json(self):
+        return {"inclusion": self.inclusion, "country_codes": self.country_codes}
+
+
+class TypeRestriction(YotiSerializable):
+    def __init__(self, inclusion, document_types):
+        self.__inclusion = inclusion
+        self.__document_types = document_types
+
+    @property
+    def inclusion(self):
+        """
+        Returns the inclusion for the type restriction
+
+        :return: the inclusion
+        :rtype: str
+        """
+        return self.__inclusion
+
+    @property
+    def document_types(self):
+        """
+        Returns the document types for the restriction
+
+        :return: the document types
+        :rtype: list[str]
+        """
+        return self.__document_types
+
+    def to_json(self):
+        return {"inclusion": self.inclusion, "document_types": self.document_types}
+
+
+class OrthogonalRestrictionsFilter(DocumentFilter):
+    def __init__(self, country_restriction, type_restriction):
+        DocumentFilter.__init__(self, filter_type=ORTHOGONAL_RESTRICTIONS)
+
+        self.__country_restriction = country_restriction
+        self.__type_restriction = type_restriction
+
+    @property
+    def country_restriction(self):
+        """
+        Returns the country restriction for the orthogonal filter
+
+        :return: the country restriction
+        :rtype: CountryRestriction
+        """
+        return self.__country_restriction
+
+    @property
+    def type_restriction(self):
+        """
+        Returns the document type restriction for the orthogonal filter]
+
+        :return: the document type restriction
+        :rtype: TypeRestriction
+        """
+        return self.__type_restriction
+
+    def to_json(self):
+        parent = DocumentFilter.to_json(self)
+        parent["country_restriction"] = self.country_restriction
+        parent["type_restriction"] = self.type_restriction
+        return parent
+
+
+class OrthogonalRestrictionsFilterBuilder(object):
+    """
+    Builder used to create an orthogonal restriction filter.
+
+    Example::
+
+        filter = (OrthogonalRestrictionsFilterBuilder()
+                  .with_whitelisted_country_codes(["GBR", "USA"])
+                  .with_whitelisted_document_types(["PASSPORT"])
+                  .build())
+
+    """
+
+    def __init__(self):
+        self.__country_restriction = None
+        self.__type_restriction = None
+
+    def with_whitelisted_country_codes(self, country_codes):
```

Comment:
```suggestion
    def with_whitelisted_countries(self, country_codes):
```

Should we use the same naming as Java? (I think both are fine and close enough)

> Same would apply elsewhere in this PR

Reply:
As discussed, we're keeping them as they are